android-interop-testing/app/build.gradle

```diff
@@ -47,21 +47,19 @@ protobuf {
             task.plugins {
                 grpc {
                     // Options added to --grpc_out
-                    option 'nano'
+                    option 'nano=true'
```

Comment:
Revert this change. It shouldn't be necessary and it was purposefully changed to `'nano'`